apps/et/et-sya-api/perftest-image-policy.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: perftest-et-sya-api
 spec:
```

Comment:
You need this on line 5

annotations:
    hmcts.github.com/prod-automated: disabled